docs/1.getting-started/10.deployment.md

```diff
@@ -64,24 +64,24 @@ module.exports = {
 
 #### Using Cluster Mode
 
-You can use `NITRO_PRESET=node_cluster` in order to leverage multi-process performance using Node.js [cluster](https://nodejs.org/dist/latest/docs/api/cluster.html) module.
+You can use `NITRO_PRESET=node_cluster` to leverage multi-process performance using Node.js [cluster](https://nodejs.org/dist/latest/docs/api/cluster.html) module.
 
-By default, the workload gets distributed to the workers with the round robin strategy.
+By default, the workload gets distributed to the workers with the round-robin strategy.
 
 ### Learn More
 
 :ReadMore{link="https://nitro.unjs.io/deploy/node" title="the Nitro documentation for node-server preset"}
 
 ## Static Hosting
 
-There are two ways to deploy a Nuxt application to any static hosting services:
+There are two ways to deploy a Nuxt application to any static hosting service:
 
-- Static site generation (SSG) with `ssr: true` pre-renders routes of your application at build time. (This is the default behaviour when running `nuxi generate`.) It will also generate `/200.html` and `/404.html` single-page app fallback pages, which can render dynamic routes or 404 errors on the client (though you may need to configure this on your static host).
-- Alternatively, you can prerender your site with `ssr: false` (static single-page app). This will produce HTML pages with an empty `<div id="__nuxt"></div>` where your Vue app would normally be rendered. You will lose many of the benefits of prerendering your site, so it is suggested instead to use `<ClientOnly>` to wrap the portions of your site that cannot be server rendered (if any).
+- Static site generation (SSG) with `ssr: true` pre-renders your application's routes at build time. (This is the default behavior when running `nuxi generate`.) It will also generate `/200.html` and `/404.html` single-page app fallback pages, which can render dynamic routes or 404 errors on the client (though you may need to configure this on your static host).
```

Comment:
Is the 's really necessary here?

Reply:
Hmm... what about entirely removing the phrase 'single-page app'? Doesn't seem to bring significant value (+ 404 error rather than errors, it's one in the end):
```suggestion
- Static Site Generation (SSG) with `ssr: true` pre-renders your application's routes at build time. (This is the default behavior when running `nuxi generate`.) It will also generate `/200.html` and `/404.html` fallback pages, which can render dynamic routes or 404 error on the client (though you may need to configure this on your static host).
```

Reply:
I think the plural is fine here, since you would need to add an 'a' for it to be singular

Reply:
Implemented alternative phrasing